bounces.go

```diff
@@ -63,7 +63,7 @@ func (m *MailgunImpl) GetBounces(limit, skip int) (int, []Bounce, error) {
 	}
 
 	r.setClient(m.Client())
-	r.setBasicAuth(basicAuthUser, m.ApiKey())
+	r.setBasicAuth(basicAuthUser, m.APIKey())
```

Comment:
Consistent with golang coding standards, but will be a breaking change for anyone who uses the library.